backend/oidc/tests/test_oidc_views.py

```diff
@@ -0,0 +1,96 @@
+import datetime
+import re
+
+import pytest
+from django.conf import settings
+from django.test import override_settings
+from django.urls import reverse
+
+from oidc.tests.factories import OIDCProfileFactory
+
+
+@pytest.mark.django_db
+@override_settings(OIDC_OP_LOGOUT_ENDPOINT="http://example.com/logout/")
+def test_logout_view(requests_mock, client, user):
+    client.force_login(user)
```

Comment:
I'd create an `user_client` fixture that already includes a user to use in these kinds of tests. Just to avoid duplicated code :)